package.json

```diff
@@ -39,6 +39,7 @@
     "sync-request": "3.0.1",
     "typescript": "1.8.7",
     "validator": "3.33.0",
-    "node-uuid": "1.4.6"
+    "node-uuid": "1.4.6",
+    "nock": "9.0.2"
```

Comment:
Then it seems like each task should be designed to be buildable on its own (in its own folder).  That is, each task should be able to specify its own devDependencies, those devDependencies should be installed during the build but only dependencies packaged during packaging.  Any specific packages the task needs should be a devDependency and not a dependency of the entire repo.  Each task is already set up to build by itself, so it's only natural that it have its own devDependencies.  Ideally, only the dependency packages get built into the VSIX (leaving devDependencies behind since they're, well, development dependencies).  Otherwise, we keep packaging up both dependencies and devDependencies and taking large packages into the VSIX (like mocha, etc.).